pkg/dashboards/istio/istio_service.go

```diff
@@ -0,0 +1,238 @@
+package istio
+
+import (
+	"github.com/perses/community-dashboards/pkg/dashboards"
+	panels "github.com/perses/community-dashboards/pkg/panels/istio"
+	"github.com/perses/community-dashboards/pkg/promql"
+	"github.com/perses/perses/go-sdk/dashboard"
+	panelgroup "github.com/perses/perses/go-sdk/panel-group"
+	listVar "github.com/perses/perses/go-sdk/variable/list-variable"
+	markdownPanel "github.com/perses/plugins/markdown/sdk/go"
+	promqlVar "github.com/perses/plugins/prometheus/sdk/go/variable/promql"
+)
+
+// General section
+func withGeneralSection(datasource string, labelMatcher promql.LabelMatcher) dashboard.Option {
```

Comment:
`datasource` is not used in this function. 
```suggestion
func withGeneralSection(labelMatcher promql.LabelMatcher) dashboard.Option {
```